### ISSUE TITLE
Plain text link name for YouTube link in Featured Content

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10301]
## Description
On the Edit screen for Vet Center, there is a component for featured National Content.  Within this area, there is a full link to a Youtube video rather than linking plain text to the video.  Screen readers will read this entire link URL which is cumbersome and difficult to understand for users.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184978166-d5527fe9-506d-4646-bb35-2fa2f45b0c22.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 2.4.4](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)

## Acceptance Criteria
- [ ] Review to determine is this can be updated at all

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by davidmpickett** [comment]
Following up on [my comment from on 16169](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16169#issuecomment-1828641808), I think that the root place that change should be implemented is on the [Display settings of the Call to Action paragraph type](https://github.com/department-of-veterans-affairs/va.gov-cms/blob/main/config/sync/core.entity_view_display.paragraph.button.default.yml). The Spotlight content paragraph type is using the default View mode for CTA paragraphs. 

If we wanted to scope this change to the Spotlight content paragraph type, we could create a new View Mode and use for when CTA paragraphs are embedded in Spotlights, but I think the intent here is to fix all instances

**Comment by davidmpickett** [comment]
Part of the underlying issue here seems to stem from a decision to separate the `Link Text` field from the `Link` field on the CTA paragraph type. The example you show of VAMC Systems is using the `Link` field to manage both the URL & the link text. That is why those links are able to show up as a unified link, because both halves of each link are stored in a single field. 

By contrast, the `Link` field on the CTA paragraph type has disabled the built in link text option.

![Screenshot 2023-11-27 155229](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/e50a2398-4312-4fbc-997a-9a1a081c38a5)

There may be a way to recombine these in the display, but honestly, the real question is why are we sometimes using the CTA paragraph type vs. the Link field, because this separation or URL and link text seems to be the main functional distinction.  🤔
